classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] @wacoo Great work adding the book class, however, you forgot to add the method for adding rentals that takes the book itself, person, and, date to create a rental, this will help accomplish the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental

Reply:
- [ ] Good job adding the method, but you are missing an important part, this method should take a parameter `date` and `person`, and then create a new instance of rental class like `Rental.new(date, self, person)` with this a has many relationship is created with rentals class